SmartDeviceLink/SDLFileManager.h

```diff
@@ -18,6 +18,10 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/// The handler that is called when the manager is set up or failed to set up with an error
```

Comment:
I think this is for internal use only, right? If so, please note that.

Reply:
added